.editorconfig

```diff
@@ -0,0 +1,7 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
Can you move this into a separate PR? I'd like to try and keep them small and focused. (I can show you some git-fu for how to do that if you'd like, because I _*totally*_ understand git :P).